pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -831,7 +831,18 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
     const enableKubernetes = K3sHelper.requiresPortForwardingFix(kubeVersion);
     const rdNetworking = !!cfg?.experimental.virtualMachine.networkingTunnel;
 
-    if (this.privilegedServiceEnabled) {
+    if (rdNetworking) {
+      guestAgentConfig = {
+        LOG_DIR:                       await this.wslify(paths.logs),
```

Comment:
`LOG_DIR`, `GUESTAGENT_KUBERNETES`, etc are all shared between all configs. Would it make sense to build the default map + overrides instead?

Or even merge `rdNetworking || this.privilegedServiceEnabled`, and just set `GUESTAGENT_PRIVILEGED_SERVICE` differently?